book/src/skip-attach.md

```diff
@@ -19,7 +19,7 @@ Specifically the `attachRequired` field instructs Kubernetes to skip any attach
 For example, the existence of the following object would cause Kubernetes to skip attach operations for all CSI Driver `testcsidriver.example.com` volumes.
 
 ```
-apiVersion: csi.storage.k8s.io/v1alpha1
+apiVersion: storage.k8s.io/v1beta1
 kind: CSIDriver
 metadata:
   name: testcsidriver.example.com
```

Comment:
Add this to the end of the doc:

## Alpha Functionality
In alpha, this feature was enabled via the [CSIDriver Object](csi-driver-object.md) CRD.

```
apiVersion: csi.storage.k8s.io/v1alpha1
kind: CSIDriver
metadata:
....
```